cmd/bootstrap/run/epochs.go

```diff
@@ -95,13 +101,17 @@ func GenerateRecoverTxArgsWithDKG(log zerolog.Logger,
 
 	// including only (conjunction):
 	//  * nodes authorized to actively participate in the current epoch (i.e. excluding ejected, joining or leaving nodes)
-	//  * and with _positive_ weight
-	currentEpochIdentities, err := snapshot.Identities(filter.And(filter.IsValidCurrentEpochParticipant, filter.HasWeightGreaterThanZero[flow.Identity]))
+	//  * with _positive_ weight,
+	//  * nodes that were not explicitly excluded
```

Comment:
```suggestion
	//  * nodes that were explicitly excluded
```

Reply:
I am a confused by your suggestion. This is the code: 
https://github.com/onflow/flow-go/blob/13468520f87bc39fcc9b05f5903184c3b0c2fdd3/cmd/bootstrap/run/epochs.go#L102-L109

and we are saying that we want to include nodes in the recovery epoch, that were _not_ explicitly blacklisted